workflows/autometa.sh

```diff
@@ -231,10 +231,10 @@ then
         set -x
         grep ">" $kingdom_fasta | \
             sed 's/^>//' | \
-            sed 's/$/_/' | \
-            cut -f1 -d" " > $orf_prefixes
+            cut -f1 -d" " | \
+            sed 's/$/_/' > $orf_prefixes
         # Retrieve ORF IDs from contig IDs
-        grep -f $orf_prefixes $orfs | sed 's/^>//' | cut -f1 -d" " > $orf_ids
+        grep -f $orf_prefixes $orfs | cut -f1 -d" " | sed 's/^>//'  > $orf_ids
```

Comment:
Why must the order of sed and cut be swapped in this line?

Reply:
Probably doesn't matter much but might be faster since there's less data being piped to the last step